WcaOnRails/config/locales/en.yml

```diff
@@ -49,6 +49,7 @@ en:
         use_wca_registration: "I would like to use the WCA website for registration"
         guests_enabled: "Guests"
         receive_registration_emails: "Notify me by email when someone registers for this competition"
+        friendly_id: "Public facing ID"
```

Comment:
Maybe something like "Public display ID"
(I'm not that sure about my suggestion, though)